templates/fm-overlay-basic.php

```diff
@@ -24,12 +27,19 @@
 				</g>
 			</svg>
 		</button>
-		<div class="fm-overlay-title">
-			<h1><?php echo esc_html( $overlay->post_title ); ?></h1>
-		</div>
-		<?php if ( '' != ( $overlay_content = get_post_meta( $overlay->ID, 'fm_overlays_content', true ) ) ) : ?>
-			<div class="fm-overlay-content">
-				<?php echo wp_kses_post( $overlay_content ); ?>
+		<?php
+		if ( ! empty( $overlay->overlay_content['content_type_select'] ) ) : ?>
+			<div class="fm-overlay-content <?php echo esc_attr( $overlay->overlay_content['content_type_select'] ); ?>">
+				<?php if ( 'image' === $overlay->overlay_content['content_type_select'] ) : ?>
+					<a href="<?php echo ( ! empty( $overlay->overlay_content['image_link'] ) ? $overlay->overlay_content['image_link'] : '' ); ?>"
+						target="<?php echo ( ! empty( $overlay->overlay_content['image_link_target'] ) ? '_blank' : '' ); ?>"
+						alt="<?php echo esc_attr( $overlay->post_title ); ?>"
+						class="fm-image-link">
+						<?php include( FM_OVERLAYS_PATH . 'templates/partials/partial-overlay-image.php' ); ?>
```

Comment:
I believe `get_template_part()` is only used within Themes [because it relies](http://wordpress.stackexchange.com/questions/124789/how-to-include-a-file-using-get-template-part-in-a-plugin/124794#124794) on the `locate_template` [function](https://codex.wordpress.org/Function_Reference/locate_template).